docs/install.rst

```diff
@@ -10,29 +10,6 @@ Installing PlasmaPy
    instructions on :ref:`installing PlasmaPy for development
    <install-plasmapy-dev>`.
 
-.. _install-requirements:
-
-Requirements
-============
-
-PlasmaPy requires Python version 3.8 or newer.
-PlasmaPy requires the following packages for installation:
-
-- `NumPy`_ — 1.18.1 or newer
-- `SciPy`_ — 1.2 or newer
-- `Astropy`_ — 4.0 or newer
-- `pandas <https://pandas.pydata.org>`_ — 1.0 or newer
-- `xarray <https://xarray.pydata.org/en/stable>`_ — above 0.14
-- `tqdm <https://tqdm.github.io>`_ — 4.41 or newer
-- `cached_property <https://pypi.org/project/cached-property>`_ — 1.5.2 or newer
-
-PlasmaPy also depends on the following packages for optional features:
-
-- `h5py <https://www.h5py.org>`_ — 2.8 or newer
-- `lmfit <https://lmfit.github.io/lmfit-py>`_ — 1.0.1 or newer
-- matplotlib_ — 2.0 or newer
-- `mpmath <https://mpmath.org>`_ — 1.0 or newer
-
```

Comment:
I am still debating about this change.  Here's my tradeoff analysis so far.

Reasons to keep this:
 - There may be some situations where users need or want to know the requirements in detail.

Reasons to delete this:
 - This information isn't necessary for users when installing PlasmaPy, and may actually be distracting.
 - This information is likely to become out-of-date when we forget to update it.
 - This is one more location where requirements are specified.
 - When advanced users need to know the requirements, they are likely to know where to look in the source code.

Alternatives
 - Update this without making other changes
 - Move this lower in the installation page (or elsewhere in the documentation)
 - Automatically get the requirements from the `.txt` files in `requirements/`

Reply:
I'd be in support of removing it, but it is scope creep.  So, I'd also support leaving it for now and taking care of it in a follow up PR.

Reply:
I'm inclined to keep it in this PR because we'd need to edit this section anyway to update the requirements even if we didn't remove it.  Thanks!

Reply:
However, I still think at a minimum we should keep the statement about Python 3.8 or newer.  It makes sense that that is stated at the beginning of the installation page.